common/src/main/java/org/opensearch/ml/common/transport/model/MLModelDeleteRequest.java

```diff
@@ -21,24 +22,40 @@
 
 import static org.opensearch.action.ValidateActions.addValidationError;
 
+@Getter
 public class MLModelDeleteRequest extends ActionRequest {
-    @Getter
     String modelId;
+    Boolean retry;
+    Integer maxRetry;
+    TimeValue retryDelay;
+    TimeValue retryTimeout;
 
     @Builder
-    public MLModelDeleteRequest(String modelId) {
+    public MLModelDeleteRequest(String modelId, Boolean retry, Integer maxRetry, TimeValue retryDelay, TimeValue retryTimeout) {
         this.modelId = modelId;
+        this.retry = retry;
+        this.maxRetry = maxRetry;
+        this.retryDelay = retryDelay;
+        this.retryTimeout = retryTimeout;
     }
 
     public MLModelDeleteRequest(StreamInput input) throws IOException {
         super(input);
         this.modelId = input.readString();
+        this.retry = input.readOptionalBoolean();
```

Comment:
Tested with old master - new data & new master - old data, no need to add version filter here to enable bwc.